incubator/minecraft/templates/secrets.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    app: {{ template "fullname" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+type: Opaque
+data:
+  rcon-password: {{ default "" .Values.MinecraftServer.Rcon.Password | b64enc | quote }}
```

Comment:
So secrets are not too secret. Known K8s challenge. Would a configmap be better?